README.md

```diff
@@ -153,18 +147,32 @@ The first, `puppetboard::apache::vhost`, will use the `apache::vhost`
 defined-type to create a full virtual host. This is useful if you want
 puppetboard to be available from http://pboard.example.com:
 
-```puppet
+(The following is generic code used in our tests, it works on Debian 9 and 10, also on Ubuntu 16.04 and 18.04. It will talk to PuppetDB on localhost via http)
 
+```puppet
 # Configure Apache on this server
-class { 'apache': }
-class { 'apache::mod::wsgi': }
+class { 'apache':
+  default_vhost => false,
+  purge_configs => true,
+}
+$wsgi = $facts['os']['family'] ? {
+  'Debian' => {package_name => "libapache2-mod-wsgi-py3", mod_path => "/usr/lib/apache2/modules/mod_wsgi.so"},
```

Comment:
This looks a lot like what I would expect from a default module path - is `mod_path` really needed?

Reply:
sadly it is. apache::mod::wsgi demands that mod_path if set if we provide package_name :(

Reply:
Too bad. At least we cleaned up the part that was hard to maintain.